draft-ietf-ace-oscore-profile.xml

```diff
@@ -788,7 +861,7 @@ OSCORE_Security_Context = {
           It is requested that IANA create a new registry entitled "OSCORE Security Context Parameters" registry.
           The registry is to be created as Expert Review Required.
           Guidelines for the experts is provided <xref target="review"/>.
-          It should be noted that in additional to the expert review, some portions of the registry require a specification, potentially on standards track, be supplied as well.
+          It should be noted that in addition to the expert review, some portions of the registry require a specification, potentially on standards track, be supplied as well.
         </t>
 
         <t>
```

Comment:
Is the name field below intended to be the JSON label or should that be specified someplace?

Reply:
yes, it should. Thanks for noting that, I missed it.

Reply:
Done in 786d6c1